### PR TITLE
Fix static test for inheritdoc

### PR DESCRIPTION
**Pull request by ravi-chandra3197**
### Description (*)

Fix static test for inheritdoc
### Fixed Issues (if relevant)

1. magento/magento2#<issue_number>: Issue title

### Manual testing scenarios (*)

1. N/A

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by orlangur** [comment]
@ravi-chandra3197 waiting for Travis builds. Are these all occurrences? Is it enforced by static tests currently?

**Comment by ravi-chandra3197** [comment]
> @ravi-chandra3197 waiting for Travis builds. Are these all occurrences? Is it enforced by static tests currently?

Hello @orlangur 
yes These occurrences in all files. And it enforced by static tests.

**Comment by orlangur** [comment]
@ravi-chandra3197 thanks, I had concerns because of `500` total number :)

File: app/code/Magento/Analytics/Model/Config/Backend/CollectionTime.php [hunk 1]
```diff
@@ -55,9 +55,9 @@ public function __construct(
     }
 
     /**
-     * {@inheritdoc}
+     * @inheritdoc
      *
-     * {@inheritdoc}. Set schedule setting for cron.
+     * @inheritdoc. Set schedule setting for cron.
```
Comment (orlangur) [inline app/code/Magento/Analytics/Model/Config/Backend/CollectionTime.php]: Incorrect change.

Comment (ravi-chandra3197) [reply app/code/Magento/Analytics/Model/Config/Backend/CollectionTime.php]: Hello @orlangur 
I have updated code can you review it.

**Comment by orlangur** [comment]
@ihor-sviziev wondering if Travis tests are working correctly if there are no new failures.

@ravi-chandra3197 we need to have a test checking there is no PHPDoc containing only `{@inheritdoc}`. Please show if it's already exists or implement one.

**Comment by ravi-chandra3197** [comment]
Hello @orlangur 
I have updated the code as per your suggestion, can you review it.

**Comment by orlangur** [comment]
@ravi-chandra3197 could you please demonstrate on a separate branch a test enforcing `{@inheritdoc}` is not present?

**Comment by ravi-chandra3197** [comment]
> @ravi-chandra3197 could you please demonstrate on a separate branch a test enforcing `{@inheritdoc}` is not present?

Hello @orlangur 
in my other pull request this commit for fix static test to remove and {@inheritdoc} and add @inheritdoc.

https://github.com/magento/magento2/pull/19806/commits/b39550ae4e947da335cea2e836005fc000b0082a

**Comment by orlangur** [comment]
@ravi-chandra3197 I don't see a failed build on Travis for your commit: https://github.com/magento/magento2/pull/19806/commits

**Comment by ravi-chandra3197** [comment]
> @ravi-chandra3197 I don't see a failed build on Travis for your commit: https://github.com/magento/magento2/pull/19806/commits

Hello @orlangur 
can you see this commit https://github.com/magento/magento2/pull/19806/commits/b39550ae4e947da335cea2e836005fc000b0082a  by p-bystritsky 
https://prnt.sc/nj0umr
And is as per Magento devdocs 
https://devdocs.magento.com/guides/v2.3/coding-standards/docblock-standard-general.html#inheritdoc

**Comment by orlangur** [comment]
@ravi-chandra3197 I'm asking you for builds results which would show errors which were fixed by @p-bystritsky.

**Comment by ravi-chandra3197** [comment]
Hello @orlangur 
I have opened my old PR for just check static test for @inheritdoc
https://github.com/magento/magento2/pull/19641
for this commit
https://github.com/magento/magento2/commit/b39550ae4e947da335cea2e836005fc000b0082a

can you review it.

**Comment by orlangur** [comment]
@ravi-chandra3197 there is no static test failed in mentioned PR. I started discussion regarding PHPDoc standard in `#app-design` Slack channel, let's put this PR on hold until discussion is finished.

**Comment by ravi-chandra3197** [comment]
> @ravi-chandra3197 there is no static test failed in mentioned PR. I started discussion regarding PHPDoc standard in `#app-design` Slack channel, let's put this PR on hold until discussion is finished.

Hello @orlangur 
Any update for this PR?

**Comment by orlangur** [comment]
Hi @ravi-chandra3197, we finally merged https://github.com/magento/devdocs/pull/4425/files#diff-bfb886455b3f21c290140e1bdbf23ed5R571 so this change is no more needed. Corresponding sniffs needs to be adjusted instead.

**Pull request closed without merging**